tidy3d/plugins/invdes/initialization.py

```diff
@@ -29,17 +32,34 @@ class RandomInitializationSpec(AbstractInitializationSpec):
 
     min_value: float = pd.Field(
         0.0,
+        ge=0.0,
+        le=1.0,
         title="Minimum Value",
         description="Minimum value for the random parameters (inclusive).",
     )
     max_value: float = pd.Field(
         1.0,
-        title="Minimum Value",
+        ge=0.0,
+        le=1.0,
+        title="Maximum Value",
         description="Maximum value for the random parameters (exclusive).",
     )
-    seed: Optional[int] = pd.Field(None, description="Seed for the random number generator.")
+    seed: Optional[pd.NonNegativeInt] = pd.Field(
+        None, description="Seed for the random number generator."
+    )
 
-    def create_parameters(self, shape: tuple[int, ...]) -> np.ndarray:
+    @pd.root_validator(pre=False)
+    def _validate_max_ge_min(cls, values):
+        """Ensure that max_value is greater than or equal to min_value."""
+        minval = values.get("min_value")
+        maxval = values.get("max_value")
+        if minval > maxval:
+            raise ValidationError(
+                f"'max_value' ({maxval}) must be greater or equal than 'min_value' ({minval})"
```

Comment:
if they are equal, it might lead to some weird behavior in the projections? might want to verify

Reply:
it's fine, it's essentially the same as uniform initialization